.teamcity/_self/projects/WebApp.kt

```diff
@@ -797,8 +797,8 @@ object RunCalypsoPlaywrightE2eTests : BuildType({
 				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
 
 				# Run the test
-				export BROWSERSIZE="mobile"
-				export BROWSERLOCALE="en"
+				export DISPLAYSIZE="mobile"
```

Comment:
I think [`viewport`](https://developer.mozilla.org/en-US/docs/Web/CSS/Viewport_concepts) instead of `display` would be a more suitable choice here since we're setting the size via the `window` object. Also, I would use `size` where we use `dimensions` and went for `name`. TL;DR: this is how I'd suggest renaming that var & related helpers:

- `BOWSERSIZE` -> `VIEWPORT_NAME`
- `getTargetDisplaySize` -> `getViewportName`
- `getDisplayResolution` -> `getViewportSize`

I didn't use the "target" keyword for the new names since I don't think it's necessary. What do you think?

Reply:
> I didn't use the "target" keyword for the new names since I don't think it's necessary

Yeah, I think that's a fair suggestion.

> I think viewport instead of display would be a more suitable choice here since we're setting the size via the window object.

Also a fair suggestion, I think. Background, I had decided on the terms as I was going off Wikipedia's definition for [display resolution](https://en.wikipedia.org/wiki/Display_resolution) and [display size](https://en.wikipedia.org/wiki/Display_size) - they appeared to suit what I was trying to describe with the terminology change. 

I'll push a change in the next revision.